cns/service/main.go

```diff
@@ -643,13 +644,19 @@ func main() {
 		}
 		defer endpointStoreLock.Unlock() // nolint
 
-		err = platform.CreateDirectory(endpointStoreLocation)
+		endpointStorePath := endpointStoreLocation
+		if runtime.GOOS == "windows" {
+			endpointStorePath = os.Getenv("CNSStoreFilePath")
```

Comment:
conventionally envvars are ALL_CAPS, and I think this should be part of the CNS config, but it looks like the Linux one is a const so it's okay for now.

Reply:
@nddq can you make a note and create a task to make this endpoint store more automatically cross-platform - we shouldn't even need to clutter up the business logic with `if GOOS` branches